.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @charlesg3 @chenwany @demartinofra @enrico-usai @francesco-giordano @gmarciani @hanwen-pcluste @lukeseawalker @notestaff @sean-smith
\ No newline at end of file
+* @charlesg3 @chenwany @demartinofra @enrico-usai @francesco-giordano @gmarciani @hanwen-pcluste @lukeseawalker @notestaff @sean-smith @dreambeyondorange
```

Comment:
This file is autogenerated and will be modified again when we rerender. We should not edit it manually.

Reply:
How do we add owners to the repo?

Reply:
Add them as maintainers and re-render the feedstock.

Reply:
Is that something any other maintainer can do?  I've asked two people and they don't know what to do

Reply:
Anyone can with the bot command: https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-rerender

Or installing conda-smithy locally.